fbchat/client.py

```diff
@@ -88,11 +88,6 @@ def __init__(self, email, password, debug=True, user_agent=None):
     def _console(self, msg):
         if self.debug: print(msg)
 
-    def _setttstamp(self):
```

Comment:
This function is only used once (in `login`), so might as well move it into `login`.